### PR TITLE
Tech/4 24/refactor auth service

### PR DESCRIPTION
**Pull request by HenadzV**
[4-24] Refactored login function using api.service

image

### CODE REVIEW

**Review by HrebenYury — APPROVED** [review]

File: exalearn-ui/src/app/pages/login-page/login-page.component.ts [hunk 1]
```diff
@@ -12,8 +12,8 @@ import { UserAuth } from '../../interfaces/interfaces';
 export class LoginPageComponent {
 	constructor(public auth: AuthService) {}
 
-	emailFormControl = new FormControl('', [Validators.required, Validators.email]);
-	passwordFormControl = new FormControl('', [Validators.required]);
+	emailFormControl = new FormControl('adminexa@mailnesia.com', [Validators.required, Validators.email]); //clear formState before prod release
+	passwordFormControl = new FormControl('_Test1234', [Validators.required]); //clear formState before prod release
```
Comment (KotdenGit) [inline exalearn-ui/src/app/pages/login-page/login-page.component.ts:17]: Are you sure you won't forget to delete them?

**Review by KotdenGit — APPROVED** [review]

File: exalearn-ui/src/app/pages/login-page/login-page.component.ts (hunk 1 above)
Comment (vstrabko) [reply exalearn-ui/src/app/pages/login-page/login-page.component.ts:17]: Please add 
// TODO 
comment here with required changes when this is not needed

**Review by vstrabko — COMMENTED** [review]
Looks good!
Just one small thing :)

**Review by YanaBr — APPROVED** [review]

File: exalearn-ui/src/app/pages/login-page/login-page.component.ts (hunk 1 above)
Comment (HenadzV) [reply exalearn-ui/src/app/pages/login-page/login-page.component.ts:17]: I won't forget. Added //TODO.

**Pull request merged**